### ISSUE TITLE
Reference for b-factor normalization

### ISSUE THREAD

**Issue by gezmi** [BioComputingUP/FLIPPER#1]
Hi,

I am looking at B-factor normalizations, I have seen such in FLIPPER but could not find any reference for the used method in the paper. I looked at Mobi2.0 which described the normalization however, without any reference. I could not yet find similar in the literature.
Did I overlook something? What is the theory behind this kind of B-factor normalization?

Thank you for the help!

**Comment by damianopiovesan** [comment]
Hi,

The mapping between the resolution and Wilson_B was taken from [here](https://strucbio.biologie.uni-konstanz.de/ccp4wiki/index.php/Wilson_plot). They cite [Popov et al. Acta Crystallogr. D. 2003](https://pubmed.ncbi.nlm.nih.gov/1283275). 

Actually we used a very conservative strategy, we set 1.0 (max high temperature) to residues with a B-factor 4 times higher than average at that resolution. Those below the threshold are scaled linearly. 

All the best
